ow_dynamic_terrain/msg/modified_terrain_diff.msg

```diff
@@ -0,0 +1,6 @@
+sensor_msgs/Image diff          # an image containing changes in height
+geometry_msgs/Point32 position  # position in the real world where the terrain
+                                # modification took place, position is the 
+                                # center of the patch
```

Comment:
I don't understand what do you mean by "position is the center of the patch", which patch?